ods_ci/tests/Resources/Page/Pipelines/OpenShiftPipelines.robot

```diff
@@ -0,0 +1,34 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+
+
+*** Variables ***
+${REDHAT_OPENSHIFT_PIPELINES_YAML}=            ods_ci/tests/Resources/Files/redhat-openshift-pipelines.yaml
+
+
+*** Keywords ***
+Data Science Pipelines Suite Setup
+    [Documentation]    Data Science Pipelines Suite Setup
+    Set Library Search Order    SeleniumLibrary
+    RHOSi Setup
+    Install Red Hat OpenShfit Pipelines
+
+Install Red Hat OpenShfit Pipelines
+    [Documentation]    Install Red Hat OpenShfit Pipelines
+    Oc Apply    kind=Subscription    src=${REDHAT_OPENSHIFT_PIPELINES_YAML}     namespace=openshift-operators
```

Comment:
if this YAML is the one used to create the pipelines operator in the cluster, I think the namespace should be `redhat-ods-operator`

Reply:
openshift-operators is correct here. This is installing the OpenShift Pipelines operator in the cluster (requirement for DAta Science Pipelines in RHODS). 

The Data Science Pipelines Operator is deployed automatically in RHODS 1.26 in redhat-ods-applications

Reply:
When I installed it for Elyra it got deployed in redhat-ods-operator

Reply:
The right namespace is `openshift-operators`. It was removed from `Oc Apply` because it is inside the file (`REDHAT_OPENSHIFT_PIPELINES_YAML`)